### DISCUSSION TITLE
What is ASSET_URL for in the config?

### DISCUSSION THREAD

**Discussion by thearchivalone** [monicahq/monica#5814] [Q&A] [answered]
My assumption is that it’s used for when you have your Cardav served through another app (Nextcloud, radicale, etc) but there’s no documentation whatsoever about it. Thanks

**Comment by asbiin** [comment, marked as answer]
This is a Laravel feature that provide a way to define a CDN host for the assets. See https://laravel.com/docs/8.x/helpers#method-asset
For your knowledge, we don't use it on Monica official instance.
